paypal/paypal-js
# Allows button message amount type to be string

Allows button message amount type to be string so that button message integration is more consistent with message integration's accepted types.


### Testing instructions:

_Setup_

- open the lighthouse-ecommerce AKA "Palmart" PR locally (message Dan for link)
- replace lines 14-18 with: 
`const initialOptions = {
        clientId: "B_Aksh3K8UpGwAs_Ngi0ahNHJja2bsxqtuqmhRAm944xzM-p6Qkq9JbqFc2yTz2Fg5WYtcW1QEu_tHDVuI",
        currency: "USD",
        intent: "capture",
        sdkBaseUrl: "https://te-msg-amt-str.qa.paypal.com/sdk/js",
        components: 'messages,buttons,marks'
};`
- run npm install
- copy paypal-js build files into Palmart node_modules
  - open this paypal-js PR in local IDE in a second window beside Palmart's IDE
  - execute npm run build in paypal-js
  - copy react-paypal-js build files
    - open paypal-js file explorer to `packages/react-paypal-js/dist` and copy the `cjs` and `esm` directories, then paste in the corresponding Palmart node_modules directory
  - copy paypal-js build files
    - open paypal-js file explorer to `packages/paypal-js/` and copy the `types` directory, then paste in the corresponding Palmart node_modules directory (will be at same level as dist)
    - open paypal-js file explorer to `packages/paypal-js/dist` and copy the `cjs` and `esm` directories, then paste in the corresponding Palmart node_modules directory
- delete .next directory (previous node_module builds are cached there, and we do not want to use those)
- execute npm run dev
  - if it does not launch and gives node: --openssl-legacy-provider is not allowed in NODE_OPTIONS, you must first set the node version to the version specified in the .nvmrc file

_Validation_

- Set the amount on the product page to equal `"200"`.  The message should still render as it did when the amount was a number.
